resources/views/welcome.blade.php

```diff
@@ -6,11 +6,13 @@
                 <!--Titles-->
                 <div
                     class="flex flex-col md:flex-row justify-start items-start w-full max-w-7xl space-y-8 md:space-y-0 md:space-x-8 mt-6">
-                    <div
-                        class="text-white w-full lg:ml-16 md:w-3/5 font-extrabold text-5xl lg:text-7xl md:text-7xl sm:text-5xl uppercase">
-                        <h1 class="leading-snug" style="text-shadow: 3px 3px 5px rgba(0, 0, 0, 0.3);">
+                    <div class="text-white w-full lg:ml-16 md:w-3/5 font-extrabold text-5xl lg:text-7xl md:text-7xl sm:text-5xl uppercase">
+                        <h1 class="leading-extra-tight" style="text-shadow: 3px 3px 5px rgba(0, 0, 0, 0.3);">
                             We are in IT together Conference
                         </h1>
+                        <h2 class="mt-5 uppercase font-bold text-sm">
+                            Powered by
+                        </h2>
```

Comment:
This should only be visible in case we have a gold sponsor (might not be an issue right now but to be kept in mind) - if the PR is merged without taken into account please open an issue